client/valor/schemas/symbolic/types.py

```diff
@@ -2124,3 +2124,64 @@ def get_type_by_name(
         return List[type_]
     else:
         raise NotImplementedError(name)
+
+
+class Text(String):
```

Comment:
Do we need tests for this and Context? I didn't see any tests for other classes in schemas/symbolic/types in the valor_client unit tests.

Reply:
For future reference, you can check this in test coverage in the github runner under  `combine-coverage-report`

Reply:
You may also be able to remove these objects by changing `Text` to `String.nullable` and `Context` to `TypedList[String].nullable` within the `Annotation` object.